MCLauncher/WUProtocol.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace MCLauncher {
     class WUProtocol {
-        private static readonly string DEFAULT_URL = "https://fe3.delivery.mp.microsoft.com/ClientWebService/client.asmx";
+        //private static readonly string DEFAULT_URL = "https://fe3.delivery.mp.microsoft.com/ClientWebService/client.asmx";
```

Comment:
This seems unrelated to the PR?

Reply:
Would you like it to be uncommented? It is unused.

Reply:
I just feel as if it should be in a separate PR as the changes do not pertain to the purpose of the PR

Reply:
I don't feel that a comment is worth an entire PR. The change is reverted.